compatibility_lib/compatibility_lib/dependency_highlighter.py

```diff
@@ -34,17 +34,17 @@ class UnstableReleaseError(Exception):
 
 
 class PriorityLevel(enum.Enum):
-    NOT_SET = "UP_TO_DATE"
-    LOW = "LOW_PRIORITY"
-    HIGH = "HIGH_PRIORITY"
+    UP_TO_DATE = 0
+    LOW = 1
+    HIGH = 2
```

Comment:
Should I preserve "LOW_PRIORITY" and "HIGH_PRIORITY" as the enum names?

Reply:
Yes that's good.